packages/core/src/test/amazonqDoc/session/sessionState.test.ts

```diff
@@ -26,4 +27,133 @@ describe('sessionStateDoc', () => {
             })
         })
     })
+
+    describe('CodeGenBase generateCode log file handling', () => {
```

Comment:
The mocking in this test code makes it difficult to tell what code paths are actually being tested here. Is all of this mocking necessary?